.github/workflows/deploy.yml

```
@@ -0,0 +1,105 @@
+          terraform_version: v1.10.3  
+      
+      - name: Change directory to Terraform directory
+        run: |
+         cd stack
+         ls -alh 
+         terraform init
+         
+        
+      - name: Terraform plan
+        run: |
+         cd stack 
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)